src/utils/common/common.tsx

```diff
@@ -10,6 +10,23 @@ export function formatDate(epochMs: number): string {
   return new Date(epochMs).toISOString().split('T')[0];
 }
 
+export function formatVerboseDate(timestamp: number | string): string {
```

Comment:
would be good to have a comment here with an example of the expected date string output; also a unit test would be good